src/spryker/docker/image/nginx/root/etc/nginx/conf.d/default.conf.twig

```diff
@@ -0,0 +1,96 @@
+
+    include snippets/certificate.conf;
+    include snippets/ssl-params.conf;
+
+    root /app/public/Yves;
+
+    # Static files - allow only specified here paths
+    # all other resources should be served via static host (and cached, if possible, via reverse proxy or cdn)
+    location ~ (/assets/|/maintenance.html|/favicon.ico|/crossdomain.xml) {
+        access_log        off;
+        expires           30d;
+        add_header Pragma public;
+        add_header Cache-Control "public";
+        try_files $uri $uri/ =404;
+        #more_clear_headers 'X-Powered-By' 'X-Store' 'X-Locale' 'X-Env' 'Server';
+    }
+
+    location ~ \.(jpg|gif|png|css|js|html|xml|ico|txt|csv|map)$ {
+        access_log off;
+        expires 30d;
+        add_header Pragma public;
+        add_header Cache-Control "public";
+        try_files $uri /index.php?$args;
+        #more_clear_headers 'X-Powered-By' 'X-Store' 'X-Locale' 'X-Env' 'Server';
+    }
+
+    # PHP application
+    location / {
+        ## CORS - Allow ajax requests from http to https webservices on the same domain
+        #more_set_headers "Access-Control-Allow-Origin: http://$server_name";
+        #more_set_headers "Access-Control-Allow-Credentials: true";
+        #more_set_headers "Access-Control-Allow-Headers: Authorization";
+
+        # Frontend - force browser to use new rendering engine
+        #more_set_headers "X-UA-Compatible: IE=Edge,chrome=1";
+
+        # Terminate OPTIONS requests immediately. No need for calling php
+        # OPTIONS is used by Ajax from http to https as a pre-flight-request
+        # see http://en.wikipedia.org/wiki/Cross-origin_resource_sharing
+        if ($request_method = OPTIONS) {
+            return 200;
+        }
+
+        add_header X-Server $hostname;
+        fastcgi_pass php-fpm:9000;
+        fastcgi_index index.php;
+        include /etc/nginx/fastcgi_params;
+        fastcgi_param SCRIPT_NAME /index.php;
+        fastcgi_param SCRIPT_FILENAME  $document_root/index.php;
+    }
+}
+
+server {
+    listen 80;
+    listen 443 ssl http2;
+
+    server_name {{ @('zed-host') }};
+
+    include snippets/certificate.conf;
+    include snippets/ssl-params.conf;
+
+    root /app/public/Zed;
+
+    # Timeout for ZED requests - 10 minutes
+    # (longer requests should be converted to jobs and executed via jenkins)
+    proxy_read_timeout 600s;
+    proxy_send_timeout 600s;
+    fastcgi_read_timeout 600s;
+    client_body_timeout 600s;
+    client_header_timeout 600s;
+    send_timeout 600s;
+
+    # Static files can be delivered directly
+    location ~ (/assets/|/favicon.ico|/robots.txt) {
+        access_log        off;
+        expires           30d;
+        add_header Pragma public;
+        add_header Cache-Control "public, must-revalidate, proxy-revalidate";
+        try_files $uri =404;
+    }
+
+    # PHP application gets all other requests
+    location / {
+        add_header X-Server $hostname;
+        fastcgi_pass php-fpm:9000;
```

Comment:
This passes every request apart from the static assets listed above to PHP, are there any access routes that should be denied or not allow PHP processing (e.g. upload directories)?

Reply:
Same as above